docs/src/basics/client-server.md

```diff
@@ -1,25 +1,48 @@
-# Client / Server
+# Architecture
 
-Zeebe uses a client / server architecture. There are two main components: *Broker* and *Client*.
+There are four main components in Zeebe's architecture: the client, the gateway, the broker, and the exporter.  
 
-![client-server](/basics/client-server.png)
+![zeebe-architecture](/basics/zeebe-architecture.png)
+
+## Client
+
+Clients are libraries that you embed in an application (e.g. a microservice that executes your business logic) to connect to a Zeebe cluster. Clients have two primary uses:
+* Carrying out business logic (starting workflow instances, publishing messages, working on tasks)
+* Handling operational issues (updating workflow instance payloads, resolving incidents)
+
+More about Zeebe clients:
+* Clients connect to the Zeebe gateway via [gRPC](https://grpc.io), which uses http/2-based transport.
+* The Zeebe project includes officially-supported Java and Go clients, and gRPC makes it possible to generate clients in a range of different programming languages. [Community clients](https://awesome.zeebe.io) have been created in other languages, including C#, Ruby, and JavaScript.
+* Client applications can be scaled up and down completely separately from Zeebe--the Zeebe brokers do not execute any business logic.
+
+## Gateway
+
+The gateway, which proxies requests to brokers, acts as a single entry point to a Zeebe cluster. This makes the cluster relatively easy to secure.
```

Comment:
> This makes the cluster relatively easy to secure

I imagine this implies you only have to secure the servers on which the gateway runs, but I wouldn't include this here in case it gives a false impression.

Reply:
@npepinpe Do you mean removing the "This makes the cluster relatively easy to secure" sentence altogether? Or that we shouldn't elaborate on it any more?

Reply:
I would remove it, personally, to avoid giving any false impression...but maybe that's just me and it looks fine to everyone else :sweat_smile:

Reply:
Makes sense--I was thinking about leaving it out from the beginning because it seems a little squishy and subjective. I removed it. Thanks.